circuit/registry_test.go

```diff
@@ -1,5 +1,3 @@
-/* +build !race */
```

Comment:
This likely never worked as expected https://pkg.go.dev/go/build#hdr-Build_Constraints